qt/widgets/common/src/FitPropertyBrowser.cpp

```diff
@@ -2830,15 +2776,6 @@ void FitPropertyBrowser::setTextPlotGuess(const QString &text) { m_displayAction
  */
 void FitPropertyBrowser::workspaceChange(const QString &wsName) {
   if (m_guessOutputName) {
-    if (isWorkspaceAGroup()) {
-      m_stringManager->setValue(m_output, QString::fromStdString(workspaceName() + "_params"));
-    } else {
-      m_stringManager->setValue(m_output, QString::fromStdString(workspaceName()));
```

Comment:
Do we still need this line because `isWorkspaceAGroup` always returns false and so this piece of code is still used?

Reply:
Looks like it, I've reviewed as it's been months since I've looked at this and it appears we can remove `m_logValue` and related functions too.

Reply:
Nice spot again with that 👍